deploy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.7
+
+RUN apk add --no-cache python3 ethtool
+
+COPY tmp/prometheus_client-0.5.0.tar.gz /usr/src/
```

Comment:
Please make this either into a docker build parameter, or use a wildcard, so that in order to use a newer version of prometheus_client library, we would only need to change it in requirements.txt.